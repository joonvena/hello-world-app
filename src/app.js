import express from 'express';
import bodyParser from 'body-parser';
import views from './views';
import version from './api/version';

const app = express();
app.use(bodyParser.json());

app.use('/api/version', version);
app.use('/', views);

app.use('/assets', express.static('src/assets'))

app.close = () => { }

export default app;