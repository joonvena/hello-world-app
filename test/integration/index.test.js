
import supertest from 'supertest';
import app from '../../src/app';

const request = supertest(app);

afterAll(() => app.close());

describe('/', () => {
  describe('GET', () => {
    it('should return page', async () => {
      const { body } = await request.get('/').expect(200);

      expect(body).toMatchSnapshot();
    });
  });
});
