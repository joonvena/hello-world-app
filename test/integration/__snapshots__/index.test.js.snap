// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ GET should return page 1`] = `Object {}`;
